sdk/tests/iot/adu/test_az_iot_adu.c

```diff
@@ -890,7 +902,7 @@ test_az_iot_adu_client_parse_service_properties_payload_no_deployment_null_file_
   assert_int_equal(request.file_urls_count, 0);
 }
 
-static void test_az_iot_adu_client_parse_update_manifest_succeed(void** state)
+static void parse_update_manifest_succeed(void** state, uint8_t request_manifest[], int32_t request_manifest_size)
 {
   (void)state;
```

Comment:
can remove this once the state is removed

Reply:
Moved back to the parent functions